sdk/src/loom/Graphics/Texture2D.ls

```diff
@@ -67,6 +84,17 @@ package loom.graphics
          * Fired when the texture is reloaded due to the backing asset changing.
          */
         public native var update:TextureInfoUpdateDelegate;
+
+        /**
+         * For asynchronous loads, this is fired when the texture has completed loading.
+         */
+        public native var asyncLoadComplete:TextureInfoAsyncLoadCompleteDelegate;
```

Comment:
Cool, will do.  For decodes, if you call texture.dispose() on a texture that is queed to load / busy loading from disk / decoding, it'll basically do a cancel, in that if it sees that flag before the load/decode, it'll just skip it and invalidate it.